recipe/meta.yaml

```diff
@@ -0,0 +1,95 @@
+{% set name = "pymoo" %}
+{% set version = "0.6.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 65637ea2a49fb836b638fd5d63a5da4f3bfa713fe8b283e0c22c5e9af3b024db
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  host:
```

Comment:
```suggestion
  build:
    - {{ compiler("c") }}
    - {{ compiler("cxx") }}
  host:
```
The C and C++ compilers is needed to compile extensions. See https://pymoo.org/installation.html#Compilation